imap_processing/tests/hi/test_utils.py

```diff
@@ -20,6 +26,76 @@ def test_hiapid():
     assert hi_apid.sensor == "90sensor"
 
 
+@pytest.mark.parametrize(
+    "test_str, compare_dict",
+    [
+        (
+            "imap_hi_l1b_45sensor-de",
+            {
+                "mission": "imap",
+                "instrument": "hi",
+                "data_level": "l1b",
+                "sensor": "45sensor",
+                "descriptor": "de",
+            },
+        ),
+        (
+            "imap_hi_l1a_hist_20250415_v001",
+            {
+                "mission": "imap",
+                "instrument": "hi",
+                "data_level": "l1a",
+                "descriptor": "hist",
+                "start_date": "20250415",
+                "version": "001",
+            },
+        ),
+        (
+            "imap_hi_l1c_90sensor-pset_20250415_v001.cdf",
```

Comment:
I should add a pointing number to this test case if the regex parsing stays.

Reply:
That would be helpful.